libs/util/shared/src/lib/elasticsearch/elasticsearch.service.ts

```diff
@@ -114,7 +115,15 @@ export class ElasticsearchService {
 
     return {
       bool: {
-        must: [{ query_string: { query } }, this.addTemplateClause('n')],
+        must: [
```

Comment:
what about the suggestions ?

Reply:
the autocomplete query is currently taken from the GN4 ui config; since there may not be a query string, I don't know how we can change the fields weight there